python/hsfs/core/feature_descriptive_statistics.py

```diff
@@ -244,3 +244,31 @@ def exact_num_distinct_values(self) -> Optional[int]:
     @property
     def start_time(self) -> Optional[int]:
         return self._start_time
+
+    @start_time.setter
+    def start_time(self, start_time: Optional[Union[datetime, date, str, int]]):
+        self._start_time = convert_event_time_to_timestamp(start_time)
+
+    @property
+    def end_time(self) -> int:
+        return self._end_time
+
+    @end_time.setter
+    def end_time(self, end_time: Optional[Union[datetime, date, str, int]]):
+        self._end_time = convert_event_time_to_timestamp(end_time)
+
+    @property
+    def row_percentage(self) -> float:
+        return self._row_percentage
+
+    @row_percentage.setter
+    def row_percentage(self, row_percentage: float):
+        if isinstance(row_percentage, int) or isinstance(row_percentage, float):
+            row_percentage = float(row_percentage)
+            if row_percentage <= 0.0 or row_percentage > 1.0:
+                raise ValueError("Row percentage must be a float between 0 and 1.")
```

Comment:
So we don't support number of rows?

Reply:
Spark does not via .sample so we will stick to percentage. Maybe change the name to row_fraction, what do you think?

Reply:
For me atleast percentage is clearer, as fraction could be based on any number/float  but percentage is based on 100 or 1 in this case.